DigitalOffice/Services/UserService/UserService.csproj

```diff
@@ -13,4 +13,8 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Controllers\" />
```

Comment:
Strange, there is a controller, but it includes a folder.

Reply:
Because we will use many controllers (maybe)